datapackage/cli_datapackage_validate.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+
+import click
+import os
+import sys
+import codecs
```

Comment:
I think `io.open` is a better choice for Py2/3 compat libs. See https://mail.python.org/pipermail/python-list/2015-March/687124.html